css/index.css

```diff
@@ -1,3 +1,382 @@
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container {
+	width: 800px;
+	margin: auto;
+	 
+}
+
+@media (min-width: 501px) and (max-width: 800px) {
+    .container {
+        width: 100%;
+    }
+}
+
+@media ( max-width: 500px ) {
+    .container {
+        width: 100%;
+        margin: 10px;
+        margin-left: 20px;
+    }
+}
+
+.nav-bar {
+	display: flex;
+	justify-content: space-around;
+	width: 800px;
+	margin: 20px auto;
+	flex-direction: row-reverse;
+	align-items: center; 
+}
+
+@media (min-width: 501px) and (max-width: 800px) {
+    .nav-bar {
+        flex-wrap: wrap;
+        flex-direction: column;
+        width: 100%;
+    }
+}
+
+@media ( max-width: 500px ) {
+    .nav-bar {
+        flex-wrap: wrap;
+        flex-direction: column;
+        /* margin: 20px auto; */
+        padding: 0;
+        width: 100%;
+        border-bottom: 1px solid black;
+    }
+}
+
+nav {
+    display: flex;
+	justify-content: space-around;
+	width: 800px;
+	margin: 20px auto;
+	flex-direction: row;
+	align-items: center; 
+}
+
+@media (min-width: 501px) and (max-width: 800px) {
+
+    nav {
+        width: 100%;
+    }
+}
+
+@media ( max-width: 500px ) {
+    
+    nav {
+        flex-direction: column;
+        justify-content: space-evenly;
+        margin-top: 20px;
+        margin-bottom: 0;
+        /* padding: 30px 0%; */
+        width: 100%;
+        text-align: center;
+
+    }
+}
+
+nav a {
+    padding: 20px auto;
+	width: 80px;
+	color: #69686e;
+    text-decoration: none;
+}
+
+@media ( max-width: 500px ) {
+    nav a {
+        padding: 20px 0;
+        width: 100%;
+        color: grey;
+        font-size: 20px;
+        border-top: 1px solid black;
+    }
+}
+
+.logo-link {
+	 margin: 0;
+}
+
+header {
+	display: flex;
+	padding-bottom: 30px;
+	border-bottom: 2px solid black;
+}
+
+@media (min-width: 501px) and (max-width: 800px) {
+    header img {
+        display: none;
+    }
+}
+
+@media ( max-width: 500px ) {
+
+    header {
+        justify-content: center;
+    }
+
+    header img {
+        display: none;
+    }
+
+}
+
+header h1 {
+	padding: 0px;
+	margin: 80px;
+	margin-bottom: 0;
+	font-size: 60px;
+	text-align: center;
+}
+
+header button {
+	margin: 20px;
+    padding: 10px 20px;
+    width: 50%;
+	border: 1px solid black;
+	background-color: white;
```

Comment:
I would try to add a `cursor: pointer`, which will make it so your button looks clickable when you hover over it.